changelog.d/find-executions-by-variable.md

```
@@ -0,0 +1,5 @@
+---
+category: Added
+---
+
+- Added the find_executions_by_variable MCP tool, which finds executions of a workflow whose input, output, or context variable matches a name (and optional value).
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)